### ISSUE TITLE
Does not work with IE10

### ISSUE THREAD

**Issue by srsgores** [doctyper/flexie#39]
At the moment, flexiejs will not work with IE10, and all elements display vertically.

**Comment by wouterj** [comment]
IE10 does support flexbox model (with a prefix). Maybe add it in the README file?

**Comment by andbits** [comment]
Does it support the old or new spec? I tried looking it up, but it seems like a hybrid of the two.

**Comment by doctyper** [comment]
IE10 supports the new flexbox model. Flexie enables the 2009 model. So these are not the droids you are looking for.

**Comment by andbits** [comment]
Yeah, I dug into it a bit more. That's nice. What tripped me up was that it supports display:-ms-flexbox instead of -ms-flex, but that part of the spec changed only very recently so I guess it's forgivable.

**Comment by cimmanon** [comment]
IE10 follows the draft from March 2012 (http://www.w3.org/TR/2012/WD-css3-flexbox-20120322/), which is not the current draft (http://www.w3.org/TR/2012/CR-css3-flexbox-20120918/).  You can see exactly which properties are supported via MSDN: http://msdn.microsoft.com/en-us/library/ie/hh772069(v=vs.85).aspx
